zgw/zaken-api/src/main/kotlin/com/ritense/zakenapi/repository/ZaakInstanceLinkRepository.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.zakenapi.repository
+
+import com.ritense.zakenapi.domain.ZaakInstanceLink
+import com.ritense.zakenapi.domain.ZaakInstanceLinkId
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import java.net.URI
+import java.util.UUID
+
+@Repository
+interface ZaakInstanceLinkRepository : JpaRepository<ZaakInstanceLink, ZaakInstanceLinkId> {
+
+    fun findByDocumentId(documentId: UUID): ZaakInstanceLink?
```

Comment:
I wonder if it will always be true there is a 1:1 relation between a zaak and a document.

Reply:
For every case in Valtimo there should only ever be one Zaak in OpenZaak